src/views/HomeView.jsx

```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import { Link } from "react-router-dom";
 import { UsersList } from "../state/use-users-list";
+import "../components/HomeViewUsers/users.css";
```

Comment:
css from components must be included ONLY by the component itself.

There should not be any cross-component imports in the codebase